### DISCUSSION TITLE
Per-file type settings?

### DISCUSSION THREAD

**Discussion by hackborn** [zed-industries/zed#6583] [Feature Requests]
Hi there, is there a way to enable a setting only for specific file types? I.e., I want my text files to word wrap, but not other file types. If this is mentioned in the docs I'm not seeing it. Thanks!

**Comment by applejag** [comment]
Maybe you've found an answer for this already, but for future devs looking for an answer to this:

```json
{
  "languages": {
    "JSON": {
      "tab_size": 4
    }
  }
}
```

Docs: https://zed.dev/docs/configuring-zed#languages

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
